.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report template
+about: Please use this template for reporting suspected bugs.
+title: 'bug: '
+labels: ''
+assignees: ''
+
+---
+
+### Issue description
+
+### Environment
+
+* APISIX Go Plugin Runner's version:
```

Comment:
I guess `APISIX version` could have something to do with bugs in the future, although currently we only got one apisix version support go plugin